lms/static/js/courseware/course_home_events.js

```diff
@@ -9,13 +9,7 @@
                 });
             });
             $('.date-summary-verified-upgrade-deadline .date-summary-link').on('click', function() {
-                Logger.log('edx.course.enrollment.upgrade.clicked', {location: 'sidebar'});
-            });
-            $('.upgrade-banner-button').on('click', function() {
-                Logger.log('edx.course.enrollment.upgrade.clicked', {location: 'notification'});
-            });
-            $('.view-verified-info').on('click', function() {
-                Logger.log('edx.course.home.learn_about_verified.clicked', {location: 'notification'});
+                Logger.log('edx.course.enrollment.upgrade.clicked', {location: 'date-sidebar'});
```

Comment:
@marcotuts @dianakhuang: I decided to rename location "sidebar" to "date-sidebar" to make it more clear, especially because the other hero might land in the sidebar, and this one might go away.

Reply:
@marcotuts: Please confirm this makes sense and is ok when you get a chance.

Reply:
Makes sense to me!